frontend/app/components/personal-log/table.js

```diff
@@ -0,0 +1,44 @@
+import Component from "@glimmer/component";
+import { inject as service } from "@ember/service";
+import { action } from "@ember/object";
+import { tracked } from "@glimmer/tracking";
+
+export default class TableComponent extends Component {
+  @service store;
+  @service router;
+  @service intl;
+  @service notify;
+
+  @tracked logs = [];
+  loaded = 0;
+
+  constructor() {
+    super(...arguments);
+    this.getLogs();
```

Comment:
We are currently doing it in here, because personal_logs ins't a component. This is the case, because I used the profile implementation as inspiration and there it's the same. Profile and Personal_logs aren't components as they don't need any logic besides showing a table. The table (ApiUser for Profile) then has the logic and makes the backend request.

Reply:
also for profile, data is fetched at route. this is the ember way of doing this.
I still think we should fetch this data inside the route